app/component/map/tile-layer/DynamicParkingLots.js

```diff
@@ -93,9 +113,9 @@ class DynamicParkingLots {
     ).then(() => {
       if (properties.free !== undefined) {
         let avail;
-        if (properties.free === 0) {
+        if (properties.free === 0 || !isOpenNow) {
           avail = 'no';
-        } else if (properties.free < 3) {
+        } else if (properties.free < 3 || isClosingSoon) {
```

Comment:
Less than 3 free is (IMHO) a bit late to indicate nearly complete state, especially if it's a large parking.  Could you please add an or-condition if percentage free is less than 10% like `properties.free/properties.capacity < 0.1 ` or so?